packages/amplify-provider-awscloudformation/src/__tests__/admin-modelgen.test.ts

```diff
@@ -79,7 +79,7 @@ it('invokes codegen functions and writes assets to S3', async () => {
 [
   [
     {
-      "Body": "mock body of mock/resource/dir/path/schema.graphql",
+      "Body": "mock body of mock/project/root/amplify-codegen-temp/schema.graphql",
```

Comment:
Notice the code block below https://github.com/aws-amplify/amplify-cli/pull/12701/files#diff-4ced5983c37a6525050acf9434abfb4c9eb6a1f99956662c7290eb8627c55841R77-R87
It seems that the `LocalPath` used for mapping to the S3 key is modified from the  original path of `schema.grapqhl` to the temp dir `${projRoot}/amplify-codegen-temp/schema.graphql`. My concern is that this might break the implicit interface between CLI and studio CMS frontend. (I am not familiar with this part so correct me if I am wrong)